content/geodesic/module/usage/with-kops.md

```diff
@@ -1,117 +1,412 @@
 ---
-title: "Using Geodesic with Kops"
+title: "Using Geodesic Module with Kops"
 description: ""
 ---
 
-{{% dialog type="info" icon="fa-info-circle" title="Prerequisites" %}}
-This assumes you've followed the [Geodesic Quick Start]({{< relref "geodesic/module/quickstart.md" >}}) guide which covers all the scaffolding necessary to get started.
+{{% dialog type="warning" icon="fa-info-circle" title="Prerequisites" %}}
+This assumes you've followed the [Geodesic Module Usage with Terraform]({{< relref "geodesic/module/usage/with-terraform.md" >}}) guide which covers all the scaffolding necessary to get started.
 {{% /dialog %}}
 
+Geodesic uses [kops]({{< relref "tools/kops.md" >}}) to manage kubernetes clusters.
+
 # Create a cluster
 
-Follow the [Provision a Cluster]({{< relref "geodesic/module/usage/with-kops.md" >}}) process
+Provisioning a `kops` cluster takes three steps:
 
-# Provision Platform Backing Services
+1. Provision a [terraform-aws-kops-state-backend]({{< relref "terraform-modules/kops-kubernetes/terraform-aws-kops-state-backend.md" >}}) which consists of an S3 bucket, cluster DNS zone, and SSH keypair to access the k8s masters and nodes.
+2. Update the `Dockerfile` and rebuild the Geodesic Module to generate a kops manifest file (and restart shell)
+3. Launch a kops cluster from the manifest file
 
-A number of [Terraform Modules Overview]({{< relref "terraform-modules/overview.md" >}}) provide to provision AWS resources needed by Charts like [external-dns](/kubernetes-backing-services/external-dns/) and [chart-repo]({{<relref "helm-charts/supported-charts/chart-repo.md" >}}). See our [Terraform modules for Kubernetes (Kops)](/terraform-modules/kops-kubernetes).
+## Provision the State Backend
 
-# Provisioning a Kops cluster
+### Config environment variables
+Add to the module `Dockerfile` environment variables
 
-We create a `kops` cluster from a manifest.
+```
+ENV KOPS_CLUSTER_NAME={KOPS_CLUSTER_NAME}
```

Comment:
I think we should use a precise example. User can figure out that it needs to be changed.

```
ENV KOPS_CLUSTER_NAME=us-west-2.example.cloudposse.org
```

This serves another purpose too: it provides our recommended naming convention. User can diverge from this convention, but then it's on them.